pkgs/top-level/python-packages.nix

```diff
@@ -25136,16 +25151,17 @@ in modules // {
   };
 
   searx = buildPythonPackage rec {
-    name = "searx-0.8.1";
+    name = "searx-${version}";
+    version = "0.9.0";
 
     src = pkgs.fetchurl {
-      url = "https://github.com/asciimoo/searx/archive/v0.8.1.tar.gz";
-      sha256 = "0z0s9n8iblrw7y5xrh2apzsazkgm4vzmxn0ckw4yfiya9am8zk32";
+      url = "https://github.com/asciimoo/searx/archive/v${version}.tar.gz";
+      sha256 = "0hbwvyah7mml452fyr7j14dqsbp1xsk7dglqpvsgg5cq0aiq9yp5";
     };
```

Comment:
Unfortunately, the current version of searx is not yet on pypi. But I'll convert that to `fetchFromGitHub`.